ultra/ultra/scenarios/generate_scenarios.py

```diff
@@ -346,7 +345,6 @@ def generate_left_turn_missions(
         }
 
     random.seed(seed)
-    np.random.seed(seed)
```

Comment:
Does removing this still allow for the scenarios to be created deterministically?

Reply:
Sorry, it does make a difference, thank you very much for questioning 😄  @christianjans